src/style.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  color: #000;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.card__photo {
+  align-self: center;
+  margin-bottom: 40px;
+  display: flex;
+  justify-content: center;
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  width: 169px;
+  height: 36px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 15px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.stars-4 {
```

Comment:
include styles for all the modifiers here (stars--1, stars--2, etc.), so that you could change the class in html and number of active stars would change accordingly